src/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Windows.cs

```diff
@@ -942,8 +942,8 @@ public static ProcessInfo[] GetProcessInfos()
 
         private static unsafe ProcessInfo[] GetProcessInfos(IntPtr dataPtr)
         {
-            // 60 is a reasonable number for processes on a normal machine.
-            Dictionary<int, ProcessInfo> processInfos = new Dictionary<int, ProcessInfo>(60);
+            // 250 is a reasonable number for processes on a normal machine.
+            var processInfos = new List<ProcessInfo>(250);
```

Comment:
On my desktop box there are ~200 so I figured this assumption is out of date.

I didn't think of the duplicate PID possibility.. I thought this was left over from some refactoring. I'll revert.